code/controllers/subsystems/persist_vr.dm

```diff
@@ -45,6 +45,19 @@ SUBSYSTEM_DEF(persist)
 		var/department_earning = J.pto_type
 		clear_unused_pto(M)
 
+		// Determine special PTO types and convert properly
+		if(department_earning == PTO_AI)
+			department_earning = PTO_CIVILIAN
```

Comment:
Whhyyyy? Why not just make PTO_AI define as PTO_CIVILIAN instead of spending the time here to check it every persist? I get why the cyborg needs logic, but this one seems silly.

Reply:
Its mix of potentially wanting AI shells to play into this and give PTO of whatever current shell is (but afterwards realizing that only civilian shells are available anyway) and that it feels like it needs some form of special handling. But since this is what it ended up being, gonna delete that now.